### PR TITLE
[ICDS Dashboard] Added number of household members in demographics tabular reports

### PR DESCRIPTION
**Pull request by challabeehyv**
https://app.asana.com/0/0/1143121160741089/f

##### SUMMARY
Added number of household members in demographics tabular reports

##### FEATURE FLAG
YES - ICDS: Enable access to pre-release features in the ICDS Dashboard reports

### CODE REVIEW

**Comment by challabeehyv** [comment]
[Demographics (1).xlsx](https://github.com/dimagi/commcare-hq/files/4353229/Demographics.1.xlsx)
with feature flag ON

**Comment by challabeehyv** [comment]
[Demographics (4).xlsx](https://github.com/dimagi/commcare-hq/files/4353231/Demographics.4.xlsx)
with feature flag OFF

File: custom/icds_reports/sqldata/exports/demographics.py [hunk 1]
```diff
@@ -349,3 +349,9 @@ def columns(self):
                 'slug': 'num_adolescent_girls_15yr18yr'
             }
         ]
+        if self.beta:
```
Comment (Rohit25negi) [inline custom/icds_reports/sqldata/exports/demographics.py]: can you add a test case for this?

**Review by Rohit25negi — CHANGES_REQUESTED** [review]

**Review by Rohit25negi — APPROVED** [review]
looks good. gtg once the test cases passed. Can you also update the ticket that this needs testing from GS team and  this is going under the feature flag for now. @challabeehyv

**Pull request merged**